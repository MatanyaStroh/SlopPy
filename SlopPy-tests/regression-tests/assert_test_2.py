# should taint both 'tokens' and 'ratio' as NA

def foo(line):
  tokens = line.split()
  ratio = float(tokens[0]) / float(tokens[1])
  assert(ratio > float(tokens[1]))
  print tokens, ratio

foo('1 2')
