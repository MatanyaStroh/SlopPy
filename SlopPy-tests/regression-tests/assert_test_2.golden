{NA <type 'exceptions.AssertionError'> AssertionError()
 Traceback (most recent call last):
  File "regression-tests/assert_test_2.py", line 6, in foo
    assert(ratio > float(tokens[1]))

Locals: {'tokens': ['1', '2'], 'line': '1 2', 'ratio': 0.5}
}
{AssertionError: Set local var 'tokens' to NA | regression-tests/assert_test_2.py foo:6}
{AssertionError: Set local var 'ratio' to NA | regression-tests/assert_test_2.py foo:6}
